tests/modules/images/dominant-color/dominant-color-test.php

```diff
@@ -108,6 +108,44 @@ public function test_tag_add_adjust_to_image_attributes( $image_path, $expected_
 		$this->assertEquals( $filtered_image_mock_lazy_load, $filtered_image_tags_not_added );
 	}
 
+	/**
+	 * Tests that dominant_color_img_tag_add_dominant_color() does not replace existing inline styles.
+	 *
+	 * @dataProvider data_provider_dominant_color_check_inline_style
+	 *
+	 * @covers ::dominant_color_img_tag_add_dominant_color
```

Comment:
```suggestion
	 * @covers ::dominant_color_img_tag_add_dominant_color
	 *
	 * @param string $filtered_image The filtered image markup.
	 *                               Must include `src="%s" width="%d" height="%d"`.
	 * @param string $expected       The expected style attribute and value.
```
Add `@param` annotations so that other contributors know exactly what should be passed to this test method.

Reply:
Sorry missed it.